src/components/internal/DataViewFilters.jsx

```diff
@@ -112,14 +112,13 @@ function DataViewFilters(props) {
 					{
 						active.map(filter => {
 							const Component = filter.component;
-							
 							return (
 								<Component
 									key={filter.name}
 									label={filter.label}
 									type={filter.type}
 									args={filter.args || {}}
-									data={props.filter[filter.name] || {}}
+									data={props.filter[filter.name] || (filter.comparisonDefault ? {comparison: filter.comparisonDefault} : {})}

```

Comment:
This approach does solve the issue for the "Titles with comparisons" filter, but now breaks the "Categories with comparisons" because this change affects all. I suggest making the change directly on the component that renders "Titles with comparisons"; I believe it's DataViewFilterText.jsx

This is the error that appears.
![image](https://user-images.githubusercontent.com/23409284/169162552-6964fafc-cd33-4e00-8f0e-048b637e4470.png)

Also remember to add some unit test to make sure the default option is the same one that renders when opening the component.